src/preferences/dialog/dlgpreferences.cpp

```diff
@@ -170,6 +173,12 @@ void DlgPreferences::createIcons() {
     m_pControlsButton->setTextAlignment(0, Qt::AlignLeft | Qt::AlignVCenter);
     m_pControlsButton->setFlags(Qt::ItemIsSelectable | Qt::ItemIsEnabled);
 
+    m_pDecksButton = new QTreeWidgetItem(contentsTreeWidget, QTreeWidgetItem::Type);
+    m_pDecksButton->setIcon(0, QIcon(":/images/preferences/ic_preferences_decks.png"));
```

Comment:
Note that this icon does not exist yet. @ronso0 or @esbrandt would you like to make it?

Reply:
Something like a Play icon should do.

![ic_preferences_decks](https://user-images.githubusercontent.com/5934199/32701107-e11f6320-c7cf-11e7-9e3f-071886ced0c8.png)

![ic_preferences_decks2](https://user-images.githubusercontent.com/5934199/32701055-60cc3d38-c7cf-11e7-8b5a-4edd6d1ad636.png)

Reply:
What's your preference? I like the boxed Play more because I asscociate it with a container rather than behaviour

Reply:
You can take them from the comment, they are not altered/compressed.